vcbuild.bat

```diff
@@ -122,6 +122,9 @@ if defined build_release (
 
 :: assign path to node_exe
 set "node_exe=%config%\node.exe"
+set "node_gyp_exe="%node_exe%" deps\npm\node_modules\node-gyp\bin\node-gyp"
+if "%target_env%"=="vs2015" set "node_gyp_exe=%node_gyp_exe% --msvs_version=2015"
+if "%target_env%"=="vs2017" set "node_gyp_exe=%node_gyp_exe% --msvs_version=2017"
```

Comment:
@joaocgreis Doesn't this only cover the case where "vs2017" or "vs2015" is passed explicitly?  Shouldn't we wait until the VS version detection is over before trying to add the argument?